### ISSUE TITLE
duplication of effort

### ISSUE THREAD

**Issue by Prince781** [matbme/tree-sitter-vala#2]
Hey,

So I had been working on the [same thing](https://github.com/Prince781/tree-sitter-vala) and I see that you added this to nvim tree-sitter before I got around to doing the same.

I haven't taken the time to look at your work but feel free to check out mine and see what you can synchronize, if anything at all.

**Comment by matbme** [comment]
Oh what a coincidence! I see you added some things in your grammar that are still missing on mine, maybe we could merge both efforts into a single repository 😃

**Comment by Prince781** [comment]
Yeah, I see you have a lot more test cases than I have, though. What I did was try my grammar out on Vala source code.
